stdlib/src/cmath.rs

```diff
@@ -78,12 +78,23 @@ mod cmath {
         z.to_complex().sin()
     }
 
+    /// Return the inverse sine of z
```

Comment:
We don't write our original document here. They are expected to be same as CPython.

Actually we don't need document anyomre. #3101 is recently brought up, so you simply can remove comment here to let it follow the default CPython document database.

Reply:
Ok. Should i also remove all the comments which were there previously?